lib/bindings/HTTPBindings.js

```diff
@@ -406,11 +408,72 @@ function start(callback) {
         router: express.Router()
     };
 
+    var options = {
+        swaggerDefinition: {
+            info: {
+                title: 'IoT Agent UL2 - HTTP', // Title (required)
+                version: '2.0.0', // Version (required)
```

Comment:
I see you changed version 1.0.0 to 2.0.0 in commit 2164dc01ef8494fe5ab99c5da6956cd8fa9b9094. However, this is not the inteded fix. 

> I think it would be a good idea to tie the version of the swagger documentation to the version of the IOTA Agent. It would be simpler.

The IOTA Agent version is taken from packages.json. Thus the solution should be something like:

```
var package = require('../../package.json');
...

info: {
...
   version: package.version,
}
```

Reply:
fixed in 2c34c9ab082a7e0b43255d21f690ffed0157703d